Loop/AppDelegate.swift

```diff
@@ -43,6 +43,8 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         
         SharedLogging.instance = deviceDataManager.loggingServicesManager
 
+        OrientationLock.deviceOrientationController = self
```

Comment:
Nit: could this cause a reference cycle?  Admittedly it should be a huge deal since AppDelegate is a singleton, but thought I'd mention it anyway.

Reply:
It's a static variable, so it has no owner. But since holding a strong reference AppDelegate is unnecessary anyway, it's actually a `static weak` variable. :)